scripts/k3s-versions.go

```diff
@@ -60,11 +60,11 @@ func getK3sChannels(ctx context.Context) (map[string]string, error) {
 	for _, channel := range channels.Data {
 		switch {
 		case channel.Name == "latest" || channel.Name == "stable":
-			break
+			// process this channel.
```

Comment:
I don't see much difference between using a `break` or adding a comment that says the case is empty.

I don't really care either way though, so still fine with the change; just curious why you think it is better.

Reply:
Because the linter complains on `break`, but it doesn't complain on a warning :)